engine/installation/linux/oracle.md

```diff
@@ -227,6 +290,11 @@ instead of `yum -y install`, and pointing to the new file.
     $ sudo rm -rf /var/lib/docker
     ```
 
+    > **Note**: This won't work when the `btrfs` graph driver has been used,
```

Comment:
"This won't work" implies that it's somehow broken instead of addressing the fact that Docker creates `btrfs` subvolumes that would need to be removed first. 

Rather, it should be something like "If you're using the `btrfs` graph driver, the Docker-created subvolumes need to be removed prior to removing the rest of the Docker content. You can review the [Oracle Linux documentation](http://docs.oracle.com/en/operating-systems/linux.html) to learn how to list and remove subvolumes from a `btrfs` volume.